tf-psa-crypto/core/psa_crypto.c

```diff
@@ -1177,11 +1183,14 @@ static psa_status_t psa_get_and_lock_transparent_key_slot_with_policy(
 
 psa_status_t psa_remove_key_data_from_memory(psa_key_slot_t *slot)
 {
+#if !defined(MBEDTLS_PSA_STATIC_KEY_SLOTS)
     if (slot->key.data != NULL) {
         mbedtls_zeroize_and_free(slot->key.data, slot->key.bytes);
```

Comment:
MAJOR: we must wipe the key data even with static key slots. If we don't, the key data is marked as unused, but it's still in memory, exposed to future compromises.